pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -123,7 +126,14 @@ assert withHomed -> withCryptsetup;
 assert withCryptsetup -> (cryptsetup != null);
 let
   wantCurl = withRemote || withImportd;
-  version = "249.7";
+  wantGcrypt = withResolved || withImportd;
+  version = "250.3";
+
+  # Bump this variable on every (major) version change. See below (in the meson options list) for why.
+  # command:
+  #  $ curl -s https://api.github.com/repos/systemd/systemd/releases/latest | \
+  #     jq '.created_at|strptime("%Y-%m-%dT%H:%M:%SZ")|mktime'
+  releaseTimestamp = "1640290180";
```

Comment:
Can we use `$SOURCE_DATE_EPOCH` instead?

Reply:
It seems we can't, as we just fetch a git tag.  `SOURCE_DATE_EPOCH` is 315532800 (1980-01-01T00:00:00+00:00).

Reply:
Let's remove the commented-out

```
# "-Dtime-epoch=1"
```

from L480 though.